shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -436,6 +458,8 @@ public Options getOptions() {
     outputFileOpt = new Option("o", "output", true, "local file to write the scan output to");
     namespaceOpt = new Option(ShellOptions.namespaceOption, "namespace", true,
         "namespace to display/set/delete properties for");
+    resourceGroupOpt = new Option(ShellOptions.resourceGroupOption, "resourceGroup", true,
```

Comment:
A bit further up is the following code that may annotate the level where something is set.  Can we work RGs into this?

https://github.com/apache/accumulo/blob/7ebaf2f3bb349cf6fe11bc8565c7484a7bd40589/shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java#L318